docs/source/guides/worklock_guide.rst

```diff
@@ -5,16 +5,11 @@ WorkLock Guide
 Overview
 --------
 
-`WorkLock` is a novel token distribution mechanism we created that requires participants to run NuCypher nodes
-to receive NU tokens. Think of it as a replacement for an ICO or an airdrop as a way to distribute initial
-tokens, but with a stronger filter for desirable participants (in our case, people who will stake and run nodes on
-the network).
-
-There will be an open contribution period in which anyone can contribute ETH to the WorkLock contract. That ETH will
-be escrowed on-chain and at the end of the contribution period, stake-locked NU will be distributed pro rata across
-contributors. If those contributors use that stake-locked NU to run a node, the NU will eventually unlock and their
-escrowed ETH will be returned in full. Alternatively, WorkLock participants can cancel their bid to forgo NU and
-recoup their escrowed ETH immediately.
+`WorkLock` is a novel, permissionless token distribution mechanism, developed at NuCypher, which requires participants to stake ETH and maintain NuCypher nodes in order to receive NU tokens.
+
+WorkLock offers specific advantages over ICO or airdrop as a distribution mechanism, chiefly: it selects for participants who are most likely to strenghten the network because they commit to staking and running nodes.
```

Comment:
```suggestion
WorkLock offers specific advantages over ICO or airdrop as a distribution mechanism, chiefly: it selects for participants who are most likely to strengthen the network because they commit to staking and running nodes.
```